neo/Consensus/ConsensusContext.cs

```diff
@@ -254,14 +254,11 @@ internal void EnsureMaxBlockSize(IEnumerable<Transaction> txs)
             txs = txs.Take((int)maxTransactionsPerBlock);
             List<UInt256> hashes = new List<UInt256>();
             Transactions = new Dictionary<UInt256, Transaction>();
-            Block.Transactions = new Transaction[0];
```

Comment:
Why remove this.. unused ?

Reply:
This was a bug, because in the previous computation of the size was used this object.